pkg/model/utils.go

```diff
@@ -121,3 +121,39 @@ func GetRepositoryURL(path string) (string, error) {
 
 	return remotes[0].Config().URLs[0], nil
 }
+
+func getDependentCyclic(s *Stack) []string {
```

Comment:
what is this for?

Reply:
This checks if the stack has any cycle in its dependencies, like 
```
services:
  app:
    depends_on:
      - db
  db:
    depends_on:
      - job
  job:
    depends_on:
      - app
```